src/intents.cs

```diff
@@ -6862,6 +6955,55 @@ interface INSetSeatSettingsInCarIntentResponse {
 		INSetSeatSettingsInCarIntentResponseCode Code { get; }
 	}
 
+	[Watch (8,0), NoTV, Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+	[BaseType (typeof(INIntent))]
```

Comment:
Probably you want a `DisableDefaultCtor` here because there is no way to set `FocusStatus` unless the provided ctor is used